<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NNLogDialog</class>
 <widget class="QDialog" name="NNLogDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>457</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="1">
      <widget class="QLineEdit" name="leOutput"/>
     </item>
     <item row="1" column="1">
      <widget class="QListWidget" name="lwInput">
       <property name="selectionMode">
        <enum>QAbstractItemView::ExtendedSelection</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Well:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Input Logs:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cbWell"/>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cbPredicted"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Predicted Log:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Hidden Neurons:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Output Log:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Training Data Ratio:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Maximum Training Epochs:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="leTrainingRatio">
       <property name="text">
        <string>0.1</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="leTrainingEpochs">
       <property name="text">
        <string>10000</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Learning Rate:</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QLineEdit" name="leLearningRate">
       <property name="text">
        <string>0.01</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="sbNeurons">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
       <property name="value">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QCheckBox" name="cbParam">
       <property name="text">
        <string>Save Parameters</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="wdParam" native="true">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLineEdit" name="leParamFile"/>
        </item>
        <item>
         <widget class="QPushButton" name="pbParamBrowse">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QCheckBox" name="cbHist">
       <property name="text">
        <string>Save History:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="wdHist" native="true">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLineEdit" name="leHist"/>
        </item>
        <item>
         <widget class="QPushButton" name="pbHistBrowse">
          <property name="text">
           <string>Browse...</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cbWell</tabstop>
  <tabstop>lwInput</tabstop>
  <tabstop>cbPredicted</tabstop>
  <tabstop>leOutput</tabstop>
  <tabstop>sbNeurons</tabstop>
  <tabstop>leTrainingRatio</tabstop>
  <tabstop>leTrainingEpochs</tabstop>
  <tabstop>leLearningRate</tabstop>
  <tabstop>cbParam</tabstop>
  <tabstop>leParamFile</tabstop>
  <tabstop>pbParamBrowse</tabstop>
  <tabstop>cbHist</tabstop>
  <tabstop>leHist</tabstop>
  <tabstop>pbHistBrowse</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NNLogDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NNLogDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbParam</sender>
   <signal>toggled(bool)</signal>
   <receiver>wdParam</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>77</x>
     <y>463</y>
    </hint>
    <hint type="destinationlabel">
     <x>265</x>
     <y>463</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbHist</sender>
   <signal>toggled(bool)</signal>
   <receiver>wdHist</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>490</y>
    </hint>
    <hint type="destinationlabel">
     <x>413</x>
     <y>490</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
